index.js

```diff
@@ -288,7 +298,7 @@ module.exports = {
 		"prefer-template": "off",
 		"quote-props": [
 			"error",
-			"as-needed"
+			"consistent"
```

Comment:
I'm less convinced on this one, and since we can't enforce a pattern it means that someone could use quotes for no reason, which we don't really want.

We could consider `consistent-as-needed`, such that if you're using "Content-Type" you'd have to use quotes for the rest, but there are cases outside of HTTP headers where quotes are necessary, and having a single property that needs to be quoted in quotes has never bothered me.